test/integration-test-main.sh

```diff
@@ -656,17 +672,19 @@ function test_update_time() {
     #
     # "touch -a" -> update ctime/atime, not update mtime
     #
-    touch -a $TEST_TEXT_FILE
-    atime=`get_atime $TEST_TEXT_FILE`
-    ctime=`get_ctime $TEST_TEXT_FILE`
-    mtime=`get_mtime $TEST_TEXT_FILE`
-    if [ $base_atime -eq $atime -o $base_ctime -eq $ctime -o $base_mtime -ne $mtime ]; then
-       echo "touch with -a option expected updated ctime: $base_ctime != $ctime, atime: $base_atime != $atime and same mtime: $base_mtime == $mtime"
-       return 1
+    if ! mount | grep s3fs | grep -e noatime -e relatime >/dev/null; then
```

Comment:
Please use `ps u $S3FS_PID | grep -q -e noatime -e relatime` which works when a user has multiple s3fs processes.

Reply:
@gaul Thanks for a comment.
ps command only knows the parameters which s3fs starts to run with.
You can find the parameters given by the OS by default in the mount command, /etc/mtab, and /proc/mounts.
Therefore, I used those commands or files to see the options that I didn't specify(or were ignored or set) at boot time.
(`cat /proc/mounts` may have been better than the mount command.)

However, it does not correspond to the state of the multiple s3fs process you pointed out, so I will fix it.
Since it cannot be determined by PID, modify it so that it is confirmed by the mount point.